.gitignore

```diff
@@ -1,6 +1,2 @@
 _internal/
 __pycache__/
-atracdenc/
-PSL1GHT/
-binmerge*
-cue2cu2*
```

Comment:
@flamming-python is it ok to remove these from your `.gitignore` file? since, after I replace each with a submodule, its presence in the `.gitignore` could interfere with proper functioning of the submodule when committing changes.